eCourse_backend/templates/registration/login.html

```diff
@@ -1,6 +1,120 @@
+{% extends "base.html" %}
+
+{% block additional-head %}
+<style>
+    input {
+        max-width: 400px;
+    }
+
+    .input-group-text {
+        width: 40px;
+    }
+    .errorIcon {
+        font-size: 2em;
+    }
+</style>
```

Comment:
If this CSS has any chance to get used in another file, please put it in a separate stylesheet

Reply:
done, should be correct now